src/DotNetWorker.Core/Invocation/IFunctionInvocationManager.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.Functions.Worker.Invocation
+{
+    /// <summary>
+    /// Keeps track of inflight invocations.
+    /// </summary>
+    interface IFunctionInvocationManager
```

Comment:
naming nit: This type is not actually dealing with the invocation as such (executing the function invocation)

Reply:
Yup, that's fair - any ideas? It's mainly for keeping track of inflight invocations so I felt like invocation needed to be in the name 🤔

Reply:
May be `FunctionInvocationDictionary` or `FunctionInvocationDetailsDictionary`? I have seen `IRpcWorkerChannelDictionary` in Host. Few other suffixes I have noticed in our repos are dict, map, resolver 😄

Reply:
I like the sound of that, thanks for the suggestion!